submodules/wizard/views/step-appRestrictions.html

```diff
@@ -0,0 +1,86 @@
+<div id="step_app_restrictions" class="wizard-step">
+{{#with i18n.accountsApp.wizard.steps.appRestrictions as |appRestrictionsI18n|}}
+	<div class="step-title">
+		{{appRestrictionsI18n.title}}
+	</div>
+	<div class="step-body">
+		<form id="form_app_restrictions" name="form_app_restrictions" action="javascript:void(0);">
```

Comment:
`name` and `action` attributes are not required by HTML5 standards, you can just go without them.
```suggestion
		<form id="form_app_restrictions">
```

Reply:
I'll just leave the `action="javascript:void(0);"`, because it prevents the browser from trying to send the form when a button is pressed and the default event behavior is not prevented via Javascript.

Reply:
Buttons should have the `type` attribute set to `button` if they are simple buttons, and set to `submit` if the goal is to "submit" the form (e.g. got to the next step for example).

This `action` attribute really is not required for our use cases.